python/django_bolt/_worker_loop.py

```
@@ -849,6 +849,30 @@ def start():
     _with_running_loop(loop, start)
 
 
+def worker_dispatch_start_cancellable(dispatch, request, loop, resolver, task_sink):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `worker_dispatch_start_cancellable`

Add return type annotation: `None`

(ANN202)